content/02.introduction.md

```diff
@@ -3,4 +3,70 @@ Set introduction for paper here
 Talk about problem, goal, and significance of paper
 
 ## Recent Work
-Talk about what has been done in the field in regards to text mining and knowledge base integration
+Relationship extraction is the process of detecting and classifying semantic relationships from a set of text.
+This process of can be broken down into three categories: (1) use of natural langue processing techniques such as manual crafted rules, dictionary lookups or text patterns to extract relationships from literature, (2) unsupervised machine learning via co-occurence scores or clustering approaches, and (3) supervised or semi-supervised machine learning using annotated datasets for classification of documents or sentences.
+We discuss previous work done for each edge type (relationship) used in this project.
```

Comment:
```suggestion
We discuss previous work done for each type of edge (relationship) studied in this project.
```

Or just introduce edge = relationship at a later point when talking about networks. Issue currently is that it makes it seem like "edge type" is synomous with "relationship", rather than with "relationship type"